### PR TITLE
Add information about morley/lorentz/indigo/cleveland

### PR DESCRIPTION
**Pull request by dcastro**
Added new sections for the following projects:

* Morley
* Lorentz
* Indigo
* Cleveland

Question for the maintainers: I see there are still some references to `morley`/`lorentz` in `files/smartcontracts/language.md`, but as far as I can tell, this file is not used to generate the actual website. It seems to be a leftover from https://gitlab.com/tezosagora/tezos-wiki/. Should I just leave that file alone?

### CODE REVIEW

File: learn/smartcontracts/README.md [hunk 1]
```diff
@@ -3,24 +3,26 @@
...
-5. [Archetype](archetype.md)
-6. [Formal Verification for Smart Contracts](michelsonandcoq.md)
+1. [SmartPy](smartpy.md)
+1. [LIGO](ligo.md)
+1. [Morley](morley.md)
+1. [Lorentz](lorentz.md)
+1. [Indigo](indigo.md)
+1. [Cleveland](cleveland.md)
+1. [Archetype](archetype.md)
+1. [Formal Verification for Smart Contracts](michelsonandcoq.md)
```
Comment (Martoon-00) [inline learn/smartcontracts/README.md:14]: Oh, this is actually a nice improvement, keeping the enumeration correct manually isn't the good way to go.

**Review by Martoon-00 — APPROVED** [review]

**Comment by pasqu4le** [comment]
cc @ilanbb1, kindly asking for a review :slightly_smiling_face:

**Review by ilanbb1 — APPROVED** [review]

**Review by ilanbb1 — COMMENTED** [review]
Looks great!

**Review by jacobarluck — APPROVED** [review]

**Comment by dcastro** [comment]
Hmm, for some reason, the 4 links under "Morley framework" on the sidebar seem to be broken.

They point to, for example, `smartcontracts/morley-framework/morley`, even though the markdown source code says it should point to `smartcontracts/morley-framework#morley`.

Screenshot from 2021-04-14 17-48-58


This work correctly locally using `gitbook serve`. The problem only seems to happen in the live website.


Do you have any idea what might have caused this @jacobarluck ?

**Pull request merged**